src/plugins/vis_type_table/public/table_vis_fn.ts

```diff
@@ -69,18 +47,18 @@ export const createTableVisFn = (): ExpressionFunctionDefinition<
   },
   fn(input, args) {
     const visConfig = args.visConfig && JSON.parse(args.visConfig);
-    const convertedData = tableVisResponseHandler(input, visConfig.dimensions);
+    const convertedData = tableVisResponseHandler(input, visConfig);
 
     return {
       type: 'render',
-      as: 'visualization',
+      as: 'table_vis',
```

Comment:
What does this do?

Reply:
I don't see any specific usage. Maybe there is something that I missed. It seems to me just an alias of data returned when use expression to fetch data from data plugin.

```
data: 
  as: "table_vis"
  params: {listenOnChange: true}
  type: "render"
  value: {visData: {…}, visType: 'table', visConfig: {…}}
```
The reason I changed it is just to make it consistent with metrics (metric_vis)and markdown (markdown_vis)